chromium/README.md

```diff
@@ -17,16 +15,17 @@ The fork, and by extension Chromium, at present only support the use of quantum-
 4. In a directory of your choosing, clone and build liboqs as follows:
 
 - `git clone --branch master https://github.com/open-quantum-safe/liboqs.git`
-- `cd liboqs && autoreconf -i`
-- `./configure --prefix=<CHROMIUM_ROOT>/third_party/boringssl/src/oqs --without-openssl --enable-shared=no`
-- `make -j && make install`
+- `cd liboqs && mkdir build && cd build`
+- `cmake .. -G"Ninja" -DCMAKE_INSTALL_PREFIX=<CHROMIUM_ROOT>/third_party/boringssl/src/oqs -DOQS_USE_OPENSSL=OFF`
+- `ninja && ninja install`
 
-5. Now, navigate to `<CHROMIUM_ROOT>` and apply the `build.gn.patch` file provided here by running `git apply <PATH_TO_PATCH_FILE>`. Then, navigate to `thid_party/boringssl`, and run `python src/util/generate_build_files.py gn`.
+5. Now, navigate to `<CHROMIUM_ROOT>` and apply the `BUILD.gn.patch` file provided here by running `git apply <PATH_TO_PATCH_FILE>`. Then, navigate to `thid_party/boringssl`, and run `python src/util/generate_build_files.py gn`.
 
-6. Finally, follow the instructions [here](https://chromium.googlesource.com/chromium/src/+/master/docs/linux/build_instructions.md#Install-additional-build-dependencies) from the "Install additional build dependencies" section onwards to build and run Chromium.
+6. Finally, follow the instructions [here](https://chromium.googlesource.com/chromium/src/+/master/docs/linux/build_instructions.md#Install-additional-build-dependencies) from the "Install additional build dependencies" section onwards to build Chromium.
 
-To verify that Chromium can perform a TLS 1.3 handshake using a post-quantum key exchange:
+To verify that Chromium can perform a TLS 1.3 handshake using a post-quantum key exchange and authentication algorithm:
 
+0. Navigate to `<CHROMIUM_ROOT>`, and start Chromium by executing `./out/Default/chrome --ignore-certificate-errors`
 1. Navigate again to the `<CHROMIUM_ROOT>/third_party/boringssl/src` folder, and build OQS-BoringSSL as a standalone project by following the instructions in that directory's `README.md` file.
-2. Then, in the `build` directory, run `./tool/bssl server -curves oqs_<KEX> -accept localhost:4433`, where `<KEX>` is any key-exchange algorithm listed [here](https://github.com/open-quantum-safe/boringssl#supported-algorithms).
-3. In the Chromium browser, load `https://localhost:4433`.
+2. Then, in the `build` directory, run `./tool/bssl server -accept 4433 -www -loop -curves <KEX>`, where `<KEX>` can be any key-exchange algorithm that are both listed [here](https://github.com/open-quantum-safe/boringssl#supported-algorithms) and are listed in the array [here](https://github.com/open-quantum-safe/boringssl/blob/master/ssl/t1_lib.cc#L375).
```

Comment:
This is pretty complex (to understand): Why check 2 locations? Am I right assuming this is again the 'manual default KEX reduction'? Wouldn't it be reasonable to _only_ look at https://github.com/open-quantum-safe/boringssl/blob/master/ssl/t1_lib.cc#L375 (it's a subset of all algorithms, right)? Also, would it be sensible to a facility such as https://github.com/open-quantum-safe/openssl#default-algorithms-announced ?

Reply:
The first location provides the names of the algorithms to pass to the `bssl server` command, and the second location lists all the algorithms that can be specified. 

I've slightly amended this sentence to make it more clear, and added a note linking to [this page](https://github.com/open-quantum-safe/boringssl/wiki/Implementation-Notes) (to be filled out) that explains why the algorithms supported by "default" are a subset of all supported algorithms.

I don't know if it's worth having an `OQS_DEFAULT_GROUPS` option for BoringSSL at the moment, as I don't think it would make building Chromium that much friendlier (the option would have to be specified somewhere around [here](https://chromium.googlesource.com/chromium/src.git/+/refs/heads/master/third_party/boringssl/BUILD.gn#38), and I haven't verified that that is all that is needed). The option might be worth having if we do start using BoringSSL for cURL.